packages/inspection-capture-web/src/PhotoCapture/PhotoCapture.tsx

```diff
@@ -103,6 +105,11 @@ export function PhotoCapture({
   const onLastSightTaken = () => {
     startTasks()
       .then(() => {
+        analytics.trackEvent('Inspection completed', {
+          inspectionId,
+          category: 'inspection_completed',
+        });
+        analytics.resetUser();
```

Comment:
Why do we reset the user at the end of the inspection?

Reply:
Actually, it was for my tests. I will remove it